pull_external.py

```diff
@@ -115,6 +165,7 @@ def _pull_files(yaml_external: dict) -> List[str]:
             )
 
     for target_dir, content in yaml_external.items():
```

Comment:
Does it makes sense here to extract the two for loops into separate methods?

Reply:
sure, it will improve readability... I'll do it 👌